재연/Programmers/Level1/64061_크레인 인형뽑기 게임.py

```diff
@@ -0,0 +1,23 @@
+def solution(board, moves):
+    from collections import deque
+    stack=[]
```

Comment:
연산 기호 사이에 공백 추가해주시면 가독성이 좋아질 것 같습니다!
```suggestion
    stack = []
```